### DISCUSSION TITLE
Original Research and Sources

### DISCUSSION THREAD

**Discussion by Ghostkeeper** [Ghostkeeper/SettingsGuide#58] [Ideas]
A lot of the articles in the Settings Guide contain what Wikipedia would call "original research". This could be seen as a bad practice. I'd like to probe the community what they think of this.

Of course, this is not a problem for things that are intrinsic to Cura or CuraEngine, like how an algorithm works or why Cura produces certain toolpaths. But there's a lot of stuff that I wrote down based on:
* Insider information working on Cura. When we have a new feature request coming in from Ultimaker or the community for a slicing feature, there's always an explanation of the reason why we need that feature. I look up what the reason was that a setting got introduced. This information is often in the commit messages and pull requests on Github, but also often in Jira (the ticket tracking system Ultimaker uses internally, which is not open to the public). And sometimes it's just in my head because I was involved in the discussions, or in meeting notes.
* My own experiences. I'm an experienced 3D printer user.
* Things I heard in the community on Reddit, Cura's Github issues, the Ultimaker community forum or other channels. These are problems people are dealing with, suggestions from the community to fix them, and often the result of those proposed fixes.

Those things are hard to use as a source.
A fourth source of information of the guide is other information online. A lot of the troubleshooting articles for instance have been written using dozens of online guides from which I've compiled the most important information, combined with the aforementioned sources.

This guide can also be seen as "expert information", precisely the sort of thing that e.g. a Wikipedia article could cite as a source. So maybe citing a source in the guide itself is not necessary?

Another option would be to increase the argumentation for this original research by including objective measurements. I wouldn't want to pollute the guide with barely-relevant information like measurement statistics on how dimensionally accurate a print is with various settings, etc. like what you would have in a scientific paper. This makes the articles hard to read. The measurements are also hardly objective then, since my printer will often be significantly different than yours.

So a couple of discussion points I'd like to propose are:
* How important do you think it is for the Settings Guide to list sources?
* How would you approach adding sources for insider information? I can't just link to a Jira ticket, since you need to be an Ultimaker employee to read them. I could link to commit messages or pull requests to Cura, but especially commit messages look very technical to the average user and don't contain a lot of information.
* Would you think it's appropriate to link to discussions in the community? It could also appear like a claim in the guide is just founded on two random people talking on the internet, while in truth it's always combined with personal experiences.
* How far do you think the guide should go about making claims verifiable?

**Comment by 5axes** [comment]
I don't think it is necessary to add more links to sources or to justify anything in your articles. The goal is to give more information about how the parameters work and in my opinion just the illustrations of the result with or without the parameter are sufficient. 

However I think that in the more detailed explanations (and why not including objective measurements) a rather long article about the method used by Cura to define **_Line Width_** would be necessary by **_comparing the Cura method with other Slicers_**. Because this point is often the origin of many errors in the tutorials and other articles concerning flow calibration that are available on the web.
